src/components/SettingsPage/Tools.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import React, { PureComponent } from 'react'
+
+import Box, { Card } from 'components/base/Box'
+import Modal, { ModalBody } from 'components/base/Modal'
+import Button from 'components/base/Button'
+import QRCodeExporter from 'components/QRCodeExporter'
+
+class TabProfile extends PureComponent<*, *> {
+  state = {
+    qrcodeMobileExportModal: false,
+  }
+
+  onQRCodeMobileExport = () => {
+    this.setState({ qrcodeMobileExportModal: true })
+  }
+
+  onRequestClose = () => {
+    this.setState({ qrcodeMobileExportModal: false })
+  }
+
+  renderQRCodeModal = ({ onClose }: *) => (
+    <ModalBody
+      onClick={onClose}
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center',
+      }}
+    >
+      <p style={{ padding: '1em' }}>
+        Open Ledger Wallet Mobile App, go to <strong>Settings {'>'} Import Accounts</strong>
```

Comment:
let's add at least a `// TODO: translate`

Reply:
is there a eslint rule to disallow every in-place texts?

Reply:
hélas no